courses/machine_learning/deepdive2/how_google_does_ml/solutions/analyze_with_bigquery_solution.ipynb

```diff
@@ -37,7 +37,7 @@
   },
```

Comment:
<p>(Per Lak) Reframe this lab, as leading to the cleaned up, deduplicated dataset so that it can be used for machine learning.</p><p>This is what you will do to create a model to predict the number of items they will order. More creating an ml dataset than data analytics.</p><p> </p><p>Maybe, explicitly state what the features would be, what we want to predict.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/615/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>